vpr/src/base/atom_netlist.cpp

```diff
@@ -188,6 +205,21 @@ AtomNetId AtomNetlist::add_net(const std::string name, AtomPinId driver, std::ve
     return Netlist::add_net(name, driver, sinks);
 }
 
+void AtomNetlist::add_net_alias(AtomNetId net_id, const std::string alias_net_name) {
+    VTR_ASSERT(net_id != AtomNetId::INVALID());
+
+    std::unordered_set<std::string> aliases;
+    auto result = net_aliases_map_.find(net_id);
+
+    if (result != net_aliases_map_.end()) {
+        aliases = result->second;
+        aliases.insert(alias_net_name);
+    } else {
+        aliases.insert(alias_net_name);
+        net_aliases_map_.insert({net_id, aliases});
+    }
+}
```

Comment:
I'm not sure this is correct. Since `aliases` is a local variable, the first branch of the if will insert `alias_net_name` into the local variable copy of the set, and is never added to `net_aliases_map_`.

I'd suggest something like:
```
net_aliases_map_[net_id].insert(alias_net_name);
```
which is much simpler and handles both cases (where net_id has been seen/not seen) directly.

Reply:
Yep, much nicer as you suggested. Done